testgen/src/header.rs

```diff
@@ -107,15 +108,16 @@ impl Generator<block::Header> for Header {
         let header = block::Header {
             version: block::header::Version { block: 0, app: 0 },
             chain_id,
-            height: block::Height(self.height.unwrap_or(1)),
+            height: block::Height::try_from(self.height.unwrap_or(1)).unwrap(), /* TODO: might
+                                                                                 * overflow */
```

Comment:
Just making sure we do not forget about this TODO.

Reply:
Covered in commit "Fixes #2".